### PR TITLE
Update django.md

### PR DESCRIPTION
**Pull request by JakeRowsell89**
Provide more information about example Dockerfile based on an answer provided in this issue:
https://github.com/docker/docker.github.io/issues/9497

### Proposed changes

Provided additional information about the Dockerfile, as suggested by a comment in related issue below.

### Related issues (optional)
Closes https://github.com/docker/docker.github.io/issues/9497

### CODE REVIEW

**Comment by JakeRowsell89** [comment]
Direct link to changed file:
https://deploy-preview-9636--docsdocker.netlify.com/compose/django/

**Review by glours — APPROVED** [review]
LTGM as it's apply @thaJeztah recommendations. 
@usha-mandya I let you validate the wording 😉

File: compose/django.md [hunk 1]
```diff
@@ -28,15 +28,22 @@ and a `docker-compose.yml` file. (You can use either a `.yml` or `.yaml` extensi
...
-       COPY . /code/
+       COPY . .
 
     This `Dockerfile` starts with a [Python 3 parent image](https://hub.docker.com/r/library/python/tags/3/).
-    The parent image is modified by adding a new `code` directory. The parent image is further modified
-    by installing the Python requirements defined in the `requirements.txt` file.
+
+    The parent image is modified by adding a new `code` directory, as well as any required intermediate directories. 
+    
+    The working directory is set to `/code`. While the `WORKDIR` keyword does create a new directory if required, it does so at root. It is therefore preferable to create a directory with `mkdir` before, which allows the container to be run as a different user.
```
Comment (usha-mandya) [inline compose/django.md:41]:
```suggestion
    The working directory is set to `/code`. While the `WORKDIR` keyword creates a new directory if required, it does so at root. Therefore, you must first create a directory using `mkdir` as this allows the container to run as a different user.
```

File: compose/django.md [hunk 2]
```diff
@@ -28,15 +28,22 @@ and a `docker-compose.yml` file. (You can use either a `.yml` or `.yaml` extensi
...
+
+    Next we `COPY` `requirements.txt` into the working directory before `RUN`-ing the `pip install -r requirements.txt` command. The order of copying `requirements.txt` (a list of dependencies to install) before running the install command is commonly used in Dockerfiles. It ensures only changes made to `requirements.txt` (or a previous line) would bust the build cache. Without changes affecting prior lines, the result of the install command can be served from the build cache, greatly speeding up the build time.
```
Comment (usha-mandya) [inline compose/django.md:43]:
```suggestion
    Next, `COPY` `requirements.txt` into the working directory before you `RUN` the `pip install -r requirements.txt` command. The order of copying `requirements.txt`, a list of dependencies to install before running the install command, is commonly used in Dockerfiles. It ensures only changes made to `requirements.txt` or a previous line would invalidate the build cache. This results in a faster build time as the install command uses build cache and doesn't have to reinstall all the dependencies when there's no change.

```

File: compose/django.md [hunk 3]
```diff
@@ -28,15 +28,22 @@ and a `docker-compose.yml` file. (You can use either a `.yml` or `.yaml` extensi
...
+    
+    Lastly we `COPY` our remaining files from our current directory to our working directory.
```
Comment (usha-mandya) [inline compose/django.md:45]:
```suggestion
    Lastly, `COPY` the remaining files from the current directory to the working directory.
```

File: compose/django.md [hunk 4]
```diff
@@ -28,15 +28,22 @@ and a `docker-compose.yml` file. (You can use either a `.yml` or `.yaml` extensi
...
+    The `COPY` of remaining files is deliberately left until the end, as it means changes to _any_ file would invalidate the cache from that command onwards.
```
Comment (usha-mandya) [inline compose/django.md:46]:
```suggestion
    The `COPY` command is deliberately included at the end, as changes to _any_ file would invalidate the cache starting with that command.
```

**Comment by usha-mandya** [comment]
@crazy-max Not sure why Netlify tests are failing with 'Failed during stage 'building site': Build script returned non-zero exit code: 1 (https://ntl.fyi/exit-code-1)`.  Here's the log - https://app.netlify.com/sites/docsdocker/deploys/63511a4faa348f0ae9b09e88 

Do you mind taking a quick look? Thanks

**Comment by craig-osterhout** [comment]
Thanks for sharing the update. Closing this PR as this content no longer exists in the docs repo.
Most of the samples have moved to https://github.com/docker/awesome-compose

**Pull request closed without merging**